### PR TITLE
added php 8.2 to ci tests

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**
Now it's broken, I think because the version of PHP is php8.2-dev

### CODE REVIEW

**Comment by fain182** [comment]
Maybe we need to change the minimum-stability? 
https://getcomposer.org/doc/04-schema.md#minimum-stability

**Comment by AlessandroMinoccheri** [comment]
We can change it but I don't know if it's better to wait for the stable release at the moment. 
We have other PRs and issues with more value I think. Do you agree @fain182 ?

File: composer.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
         "vimeo/psalm": "^4.6",
         "friendsofphp/php-cs-fixer": "^2.16",
         "phpunit/phpunit": "^7.5|^9.0",
-        "mikey179/vfsstream": "^1.6"
+        "mikey179/vfsstream": "^1.6",
+        "phpspec/prophecy": "^1.10"
```
Comment (pfazzi) [inline composer.json:42]: why?

**Review by pfazzi — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (fain182) [reply composer.json:42]: I suppose this error was the reason 
https://github.com/phparkitect/arkitect/runs/8292607729?check_suite_focus=true

**Pull request closed without merging**